modules/gcp-passthrough-nlb/variables.tf

```diff
@@ -0,0 +1,103 @@
+### Variables
+
+variable "region" {
+  default = "us-central1"
+}
+
+variable "zone" {
+  default = "us-central1-a"
+}
+
+variable "project_id" {
+  type = string
+  validation {
+    condition     = can(regex("^([a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$", var.project_id))
+    error_message = "must be a valid project name"
+  }
+}
+
+variable "network_id" {
+  validation {
+    condition     = can(regex("^([a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$", var.network_id))
+    error_message = "must be a valid network name"
+  }
+}
+
+variable "compute_instance_name" {
+  type = string
+  validation {
+    condition     = can(regex("^([a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$", var.compute_instance_name))
+    error_message = "must be a valid instance name"
+  }
+}
+
+variable "compute_instance_ip_address" {
+  type = string
+}
+
+variable "compute_instance_network_tags" {
+  type = list(string)
+}
+
+variable "compute_instance_subnetwork" {
+  type        = string
+  description = "the subnetwork the instance is in"
+  validation {
+    condition     = can(regex("^([a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$", var.compute_instance_subnetwork))
+    error_message = "must be a valid subnetwork name"
+  }
+}
+
+
+variable "name" {
+  default = "sourcegraph-psc"
+}
+
+variable "healthcheck-https-path" {
+  default = "/status"
+}
+
+variable "domain" {
+  type    = string
+  default = null
+}
```

Comment:
it looks like GHE does not care about Host header in order to make health check work, we can remove this?